lsp/server/src/server.ts

```diff
@@ -155,6 +141,58 @@ connection.languages.diagnostics.on(async (params) => {
     }
 });
 
+connection.onCodeAction((params) => {
+    const textDocument = documentCache.get(params.textDocument.uri);
+    const diagnostics = params.context.diagnostics;
+	if (textDocument === undefined || diagnostics.length === 0) {
+		return undefined;
+	}
+    
+	const result: CodeAction[] = [];
+    diagnostics.forEach((item) => {
+        const { data } = item;
+        if (data !== undefined) {
+            const metData = data as DiagnosticMetaData;
+            if (metData.producerId !== undefined) {
+
+                const suppressedIds = new Set(settings.diagnostic.suppressedIds);
+                suppressedIds.add(metData.producerId);
+                const suppressThisDiagnostic: CodeAction = {
+                    title: `Suppress such diagnostic: ${metData.producerId}`, 
```

Comment:
the action action title is visible to developer like below, need a official wording for it.
![image](https://github.com/user-attachments/assets/30366e58-d15e-4b92-bc83-936697d51a4d)